solutions/usaco-831.mdx

```diff
@@ -2,15 +2,117 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Kevin Sheng, Benjamin Qi, Brad Ma
+author: Jesse Choe, Kevin Sheng, Benjamin Qi, Brad Ma, Aadit Ambadkar
 ---
 
 ## Solution 1
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_tttt_bronze_open18.html)
 
 <LanguageSection>
+<CPPSection>
 	
+```cpp
+#include <bits/stdc++.h>
```

Comment:
I don't see a need to include this; it's not more readable than the original code in the analysis.

Reply:
^